signingscript/docker.d/init_worker.sh

```diff
@@ -29,22 +29,30 @@ export GPG_PUBKEY_PATH=$APP_DIR/signingscript/src/signingscript/data/gpg_pubkey_
 export WIDEVINE_CERT_PATH=$CONFIG_DIR/widevine.crt
 export AUTHENTICODE_TIMESTAMP_STYLE=old
 export AUTHENTICODE_TIMESTAMP_URL=http://timestamp.digicert.com
-export AUTHENTICODE_CERT_PATH_202404=$APP_DIR/signingscript/src/signingscript/data/authenticode_dep.crt
-export AUTHENTICODE_CERT_PATH="$AUTHENTICODE_CERT_PATH_202404"
+export AUTHENTICODE_CERT_PATH_202412=$APP_DIR/signingscript/src/signingscript/data/authenticode_dep.crt
+export AUTHENTICODE_CERT_PATH_202404="$AUTHENTICODE_CERT_PATH_202412"
+export AUTHENTICODE_CERT_PATH="$AUTHENTICODE_CERT_PATH_202412"
 export AUTHENTICODE_CERT_PATH_EV="$AUTHENTICODE_CERT_PATH"
-export AUTHENTICODE_CA_PATH_202404=$APP_DIR/signingscript/src/signingscript/data/authenticode_dep_ca.crt
-export AUTHENTICODE_CA_PATH="$AUTHENTICODE_CA_PATH_202404"
+export AUTHENTICODE_CERT_PATH_EV_202412="$AUTHENTICODE_CERT_PATH"
+export AUTHENTICODE_CA_PATH_202412=$APP_DIR/signingscript/src/signingscript/data/authenticode_dep_ca.crt
+export AUTHENTICODE_CA_PATH_202404="$AUTHENTICODE_CA_PATH_202412"
+export AUTHENTICODE_CA_PATH="$AUTHENTICODE_CA_PATH_202412"
 export AUTHENTICODE_CA_PATH_EV="$AUTHENTICODE_CA_PATH"
+export AUTHENTICODE_CA_PATH_EV_202412="$AUTHENTICODE_CA_PATH"
 export AUTHENTICODE_CA_TIMESTAMP_PATH=/usr/lib/ssl/certs/ca-certificates.crt
 export AUTHENTICODE_CROSS_CERT_PATH=$APP_DIR/signingscript/src/signingscript/data/authenticode_stub.crt
 if [ "$ENV" == "prod" ]; then
   export AUTHENTICODE_TIMESTAMP_STYLE=old
+  export AUTHENTICODE_CERT_PATH_202412=$APP_DIR/signingscript/src/signingscript/data/authenticode_prod_202412.crt
   export AUTHENTICODE_CERT_PATH_202404=$APP_DIR/signingscript/src/signingscript/data/authenticode_prod_202404.crt
   export AUTHENTICODE_CERT_PATH="$AUTHENTICODE_CERT_PATH_202404"
   export AUTHENTICODE_CERT_PATH_EV=$APP_DIR/signingscript/src/signingscript/data/authenticode_prod_ev.crt
-  export AUTHENTICODE_CA_PATH_202404=$APP_DIR/signingscript/src/signingscript/data/authenticode_prod_ca_202404.crt
-  export AUTHENTICODE_CA_PATH="$AUTHENTICODE_CA_PATH_202404"
-  export AUTHENTICODE_CA_PATH_EV=$APP_DIR/signingscript/src/signingscript/data/authenticode_prod_ev.crt
+  export AUTHENTICODE_CERT_PATH_EV_202412=$APP_DIR/signingscript/src/signingscript/data/authenticode_prod_ev_202412.crt
+  export AUTHENTICODE_CA_PATH_202412=$APP_DIR/signingscript/src/signingscript/data/authenticode_prod_ca_202412.crt
```

Comment:
This file doesn't seem to exist, should we point at `authenticode_prod_ca_202404.crt` instead?  Or did you mean to rename `_202404` to `_202412`?

Reply:
Thanks for catching that.

To be honest, I'm not sure what I intended here. With fresh eyes though, it seems like removing the date is the best option seeing as there's only one version of it after https://github.com/mozilla-releng/scriptworker-scripts/commit/e4fc7ac12f31e44f5e25e1da7edf5e568d3388ad.